fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBMetaDataStore.java

```diff
@@ -319,13 +367,33 @@ protected RecordMetaData buildMetaData(RecordMetaDataProto.MetaData metaDataProt
      */
     @Nonnull
     public CompletableFuture<Void> preloadMetaData(@Nullable RecordMetaDataProvider metaDataProvider) {
+        Descriptors.FileDescriptor localFileDescriptor = metaDataProvider == null ? null :
+                                                                                    metaDataProvider.getRecordMetaData().getRecordsDescriptor();
```

Comment:
Hm, I think this is a...sensible enough default. It kind of assumes that getting the meta-data from the seed provider is cheap, but maybe that's a fine enough assumption.

As it is, this is the *only* way to specify a seed descriptor from an `FDBRecordStore` builder, which I don't think we want.